### DISCUSSION TITLE
请问英文封面的提交日期在哪里修改

### DISCUSSION THREAD

**Discussion by ppppls115** [note286/xduts#169] [Q&A] [answered]
我把info/submit-date设置为2024-4，中文封面有月份，英文封面没有月份，只有2024

**Comment by note286** [comment, marked as answer]
```latex
\documentclass{xdupgthesis}
\xdusetup{info/submit-date={2024-4}}
\begin{document}
\end{document}
```

![image](https://github.com/note286/xduts/assets/16207292/c7491c83-c1eb-49df-b820-b196065ee5af)

**Reply by ppppls115** [reply]
感谢回复！可以了，之前日期是拿三角符号括起来的，换成大括号就好了

**Reply by note286** [reply]
你对`<>`有些误解，举个例子，`<日期>`和`2024-04`等价，再或者，`<title>`和`研究小行星爆炸`等价。如果不使用`<>`来保护，那么真的会有人就写个`title`字样的。你现在应该明白什么意思了。

**Reply by ppppls115** [reply]
噢，明白了。填的时候都不带三角括号呗。

**Reply by sikouhjw** [reply]
> 噢，明白了。填的时候都不带三角括号呗。

解释了还是跟没解释一样……

**Reply by TIAN-CUTE** [reply]
> 你对`<>`有些误解，举个例子，`<日期>`和`2024-04`等价，再或者，`<title>`和`研究小行星爆炸`等价。如果不使用`<>`来保护，那么真的会有人就写个`title`字样的。你现在应该明白什么意思了。

懂了 ~v~
